sqllogictest/src/parser.rs

```diff
@@ -272,14 +280,19 @@ fn parse_inner(loc: &Location, script: &str) -> Result<Vec<Record>, ParseError>
             }
             ["statement", res @ ..] => {
                 let mut expected_count = None;
-                let error = match res {
-                    ["ok"] => false,
-                    ["error"] => true,
+                let mut expected_error = None;
+                match res {
+                    ["ok"] => {}
+                    ["error", err_str @ ..] => {
+                        let err_str = err_str.join(" ");
```

Comment:
I guess this won't always work, since we do `line.split_whitespace()`. 😅 But I don't bother fixing it now.